client/actions/documentsActions.js

```diff
@@ -0,0 +1,200 @@
+import axios from 'axios';
+import { notify } from 'react-notify-toast';
+import { browserHistory } from 'react-router';
+import { DISPLAY_ALL_DOCUMENTS, DISPLAY_MY_DOCUMENTS, UPDATE_DOCUMENT_SUCCESS,
```

Comment:
This import list is growing, you can use ` import * as types from '../actions/types'` then you can do `types.DISPLAY_ALL_DOCUMENTS`